apps/ios/Shared/Views/Chat/ChatView.swift

```diff
@@ -586,7 +601,42 @@ struct ChatView: View {
                 }
             }
         }
-        
+
+        private func reactionUIMenu() -> UIMenu? {
+            let rs = MsgReaction.values.compactMap { r in
+                ci.reactions.contains(where: { $0.reaction == r })
+                ? nil
+                : UIAction(title: r.text) { _ in setReaction(r, add: true) }
```

Comment:
it's counterintuitive that reaction is not in the list when it's on chat item and the user didn't react with it. ok to remove from the list if the user used it

Reply:
maybe don't remove them from this list at all, and instead disable reactions made by user?